upi/openstack/update_ign.py

```diff
@@ -10,18 +11,22 @@
 #According to the number of the master count, create the JSON files, and add hostname/network-scripts.
 #According to the number of the worker count, create the JSON files, and add hostname/network-scripts.
 
-# Read inventory.yaml for CiscoACI CNI variables
-with open("inventory.yaml", 'r') as stream:
+# Read inventory.yaml for CiscoACI CNI variable
+original_inventory = "inventory.yaml"
+processed_inventory = "inventory.yaml"
```

Comment:
Just a trivial comment, if the name is the same, and nothing else here do we need an extra variable here? because there is no difference here

Reply:
Modified accordingly, keeping the two variables though in case we don't want to change the user-created inventory